File: Hero/Models/TemplateUploadResult.cs
using Hero.Models.Validation;
using System.Collections.Generic;
using WebApiDto.Dto;

namespace Hero.Models
{
    public class TemplateUploadResult
    {
        public List<PropertyLimit> PropertyLimits { get; set; }
        public PropertyLimitFloatingValues PropertyLimitFloatingValues { get; set; }
        public List<ValidationResult> ValidationResults { get; set; }
        public int? FirstLossLimitValue { get; set; }
        public string? TemplateValidationError { get; set; }
    }
}

File: Hero/ClientApp/src/app/models/template-upload-result.ts
import { PropertyLimit } from '@app/quote/models/property-limit.model';
import { ValidationResult } from './validation-result';
import { PropertyLimitFloatingValues } from '@app/quote/models/property-limit-floating-values.model';

export class TemplateUploadResult {
    public propertyLimits: PropertyLimit[];
    public validationResults: ValidationResult[];
    public propertyLimitFloatingValues?:PropertyLimitFloatingValues;
    public firstLossLimitValue?: number;
    public templateValidationError?: string;
}
